docs/guide/node_in_cloud.md

```diff
@@ -0,0 +1,152 @@
+# Deploy tfjs-node project on cloud platform
+
+This doc describes how to run a Node.js process with @tensorflow/tfjs-node package on cloud platforms.
+
+Starting from tfjs-node@1.2.4, running Node.js project on cloud platforms does not require extra configurations. This guide will use the [mnist-node](https://github.com/tensorflow/tfjs-examples/tree/master/mnist-node) example in @tensorflow/tfjs-examples repository, which trains MNIST with tfjs-node package on Heroku and GCloud. The details of Heroku’s Node.js Support can be found in [Heroku Node.js Support](https://devcenter.heroku.com/articles/nodejs-support) article. The details of Node.js on Google Cloud Platform can be found on the [Google Cloud page](https://cloud.google.com/nodejs/docs/).
+
+## Deploy Node.js project on Heroku
+
+### Prerequisites
+
+1. Node.js and npm installed
+2. Heroku account
+3. Heroku CLI
+
+### Create the Node.js app
+
+1. Create a folder and copy the `data.js`, `main.js`, `model.js` and `package.json` files from the [mnist-node](https://github.com/tensorflow/tfjs-examples/tree/master/mnist-node) example.
+2. Make sure the @tensorflow/tfjs-node dependency is @1.2.4 or newer version.
+
+### Build your app and run it locally
+
+1. Run the `npm install` command in your local directory to install the dependencies that are declared in the `package.json` file. You should be able to see the tfjs-node package is installed and libtensorflow is downloaded.
```

Comment:
"... able to see that the tfjs-node package ...".

Reply:
Done.